docker-services.yml

```diff
@@ -43,10 +43,9 @@ services:
   db:
     image: postgres:9.6
     restart: "always"
-    environment:
-      - "POSTGRES_USER=cd2h-repo-project"
-      - "POSTGRES_PASSWORD=cd2h-repo-project"
-      - "POSTGRES_DB=cd2h-repo-project"
```

Comment:
Should we update `postgres/init-app-db.sh` as well?

Reply:
That is a very astute remark. After 1h30 of investigation, it turns out that `docker/postgres/init-app-db.sh` is not used actually. I suspect it is either an artefact or it is a suggestion for production usage. Either way, I am removing it, because it is confusing.